sqlcl/format.js

```diff
@@ -372,8 +409,12 @@ var run = function(args) {
         if (options.rootPath == "*") {
             formatBuffer(formatter);
         } else {
-            var files = getFiles(options.rootPath, options.extensions);
-            formatFiles(files, formatter, options.markdownExtensions);
+            if (options.files) {
```

Comment:
should be changed to `if (options.files.length > 0) {`

Reply:
Good catch. Fixed.